lib/mixins/element-mixin.js

```diff
@@ -466,7 +470,8 @@ export const ElementMixin = dedupingMixin(base => {
           this.prototype.hasOwnProperty(JSCompiler_renameProperty('_template', this.prototype)) ?
           this.prototype._template :
           // Look in dom-module associated with this element's is
-          (getTemplateFromDomModule(/** @type {PolymerElementConstructor}*/ (this).is) ||
+          ((this.hasOwnProperty(JSCompiler_renameProperty('is', this)) &&
```

Comment:
Why is this change needed? It may be better in general, but it's unclear why it's required.

Reply:
OK, it's so that an `is` below this prototype cannot override `_template` below this prototype.